doc/man1/flux-jobs.rst

```diff
@@ -78,6 +78,12 @@ OPTIONS
    option is used, then each child instance in the hierararchy is listed
    with its stats.
 
+**--recurse-all**
+   By default, jobs not owned by the user running ``flux jobs`` are
+   skipped with ``-R, --recurseive``. This option forces the command
+   to attempt to recurse into the jobs of other users.  Implies
+   ``--recursive``.
+
```

Comment:
Maybe add a note that "By default, Flux instances can only permit the instance owner to connect"?

Reply:
Great idea! I added that comment, rebased on current master and pushed the result.